source/grid/tria.cc

```diff
@@ -8995,28 +8995,79 @@ Triangulation (const Triangulation<dim, spacedim> &other)
 
 
 
+#ifdef DEAL_II_WITH_CXX11
+template <int dim, int spacedim>
+Triangulation<dim, spacedim>::
+Triangulation (Triangulation<dim, spacedim> &&tria)
+  :
+  Subscriptor(static_cast<Subscriptor&&>(tria)),
+  smooth_grid(tria.smooth_grid),
+  periodic_face_pairs_level_0(tria.periodic_face_pairs_level_0),
+  periodic_face_map(tria.periodic_face_map),
+  levels(tria.levels),
+  faces(tria.faces),
+  vertices(std::move(tria.vertices)),
+  vertices_used(tria.vertices_used),
```

Comment:
I think [this SO answer](http://stackoverflow.com/a/14486705/1762306) should clarify. It's very confusing.